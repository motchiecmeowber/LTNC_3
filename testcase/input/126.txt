BEGIN
INSERT x1 string
INSERT x2 string
BEGIN
INSERT x3 string
ASSIGN x3 x2
ASSIGN x2 x1
BEGIN
LOOKUP x3
END
LOOKUP x2
PRINT
END
LOOKUP x1
END
RPRINT