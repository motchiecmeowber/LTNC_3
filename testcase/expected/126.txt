success
success
success
success
success
2
1
x1//1 x2//1 x3//2
1
